module2/intermission_work/index.md

```diff
@@ -1,5 +1,62 @@
 ---
+title: Mod 2 Intermission Work
 layout: page
-title: Module 2 - Intermission Pre-Work
 ---
 
+## Learning Goals
+
+* Explore and sharpen JavaScript fundamentals
+* Understand basic HTML elements and attributes
+* Understand basic CSS styling
+* Understand what a relational database is
+* Interact with a relational database by writing SQL
+* Explore the basics of Web Development
+  * Understand and Implement CRUD functionality
+  * Understand and implement the MVC design pattern
+* Begin learning the conventions of a Rails app
+
+## Deliverables:
+
+First, create a new [Gist on Github](https://gist.github.com/). Copy and paste the markdown from [this template](https://gist.github.com/mikedao/cf0f63490a0ef91ac7d251e95fc410f7) into your gist.
+
+This Gist will be your submission for your intermission work. Each of the assignments includes a Check for Understanding that you must complete in your Gist. Some of the assignments will also ask you to include a link to a GitHub repo.
+
+**Deliverables are due 5:00 pm the Saturday before the module begins. Prework must be completed in its entirety in order to be successful in Mod 2.**
+
+## Assignments
+
+Work on the following assignments in order. Each assignment will assume that you have completed the previous one and the accompanying Checks for Understanding. **Ruby 3.2.2 and Rails 7.1.2 are required.**
+
+Time-box recommendations are in each section's instructions, and also included here for convenience. 
+
+1. Sign up for the [GitHub Student Developer Pack](https://education.github.com/pack), using your `@turing.edu` email address. 
+   - This step sometimes takes a few days for GitHub to complete, but once you sign up for your credits you'll be able to take advantage of their benefits! 
+   - Follow [these instructions](https://mod4.turing.edu/lessons/intermission/gh-student-instructions.html) to get the info you require to complete registration. **You do NOT need to complete the Heroku steps at this time!** Ask in #codehelp if you get stuck. 
+2. [Ruby and Rails versions installation guide](./ruby_and_rails_versions.md)
+3. [Get Module 2 Tools](./tools)
+4. [Intro to LinkedIn](./linkedin)
+5. Styling (4 hrs total)
+    * [Intro to HTML](./html) (2h)
+    * [Intro to CSS](./css) (2h)
+6. [Intro to SQL](./sql) (2 hrs total)
+7. Complete this [Intro to APIs in Ruby on Rails](./intro_to_rails)  (30min - 1h)
```

Comment:
Per question in Slack, I feel 30-60 minutes is probably not substantial enough to be worth their while since they won't be using this knowledge for another 2-3 weeks. I vote remove or move to Optional.